blackjax/mcmc/integrators.py

```diff
@@ -49,16 +49,17 @@ def velocity_verlet(
 ) -> EuclideanIntegrator:
     """The velocity Verlet (or Verlet-Störmer) integrator.
 
-    The velocity Verlet is a two-stage palindromic integrator [1]_ of the form
+    The velocity Verlet is a two-stage palindromic integrator [BouRabee2018]_ of the form
```

Comment:
From here on it start to differ than above in terms of reference style (`[1]`), we should keep it consistent.

Reply:
Just saw https://github.com/blackjax-devs/blackjax/issues/493, so we should use explicit ref everywhere right?

Reply:
I think we should, it will make #473 easier. I would even change the citation style to follow what Sphinx recommends.

Reply:
Let's uniformize all this when solving #493.